src/backend/input.rs

```diff
@@ -216,6 +231,9 @@ pub trait InputBackend: Sized {
     /// Get current `InputConfig`
     fn input_config(&mut self) -> &mut Self::InputConfig;
 
+    /// Called to inform the Input backend about a new focused Output for a `Seat`
+    fn set_output_metadata(&mut self, seat: &Seat, output: &Output);
```

Comment:
If `Output` is a trait, can I'd rather have this method generic to avoid virtual dispatch.

But couldn't this method just be `fn set_output_metadata(&mut self, width: u32, height: u32)` ?

Also, the question of device giving absolute positions (such as touchscreens) and locked to one output, independently of pointer focus remains.